<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="jenkins-agent-10-25-196-175" failures="0" tests="6" name="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" time="60.399" errors="0" timestamp="2023-09-07T17:12:17 UTC" skipped="0">
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_504" time="2.065"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_429_And_No_MaxRetryTime" time="0.097"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_502" time="13.057"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_429" time="5.072"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_504_And_No_MaxRetryTime" time="0.051"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_503" time="40.057"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest -->
